src/store/peer/reducer.ts

```diff
@@ -55,12 +66,24 @@ const deletedPeer = createReducer<DeleteResponse<string | null>, ActionTypes>(in
     .handleAction(actions.deletedPeer.success, (store, action) => action.payload)
     .handleAction(actions.deletedPeer.failure, (store, action) => action.payload)
     .handleAction(actions.setDeletePeer, (store, action) => action.payload)
+    .handleAction(actions.resetDeletedPeer, () => initialState.deletedPeer)
+
+const updateGroupsVisible = createReducer<boolean, ActionTypes>(initialState.setUpdateGroupsVisible)
+    .handleAction(actions.setUpdateGroupsVisible, (store, action) => action.payload)
+
+const savedGroups = createReducer<ChangeResponse<Group[] | null>, ActionTypes>(initialState.savedGroups)
+    .handleAction(actions.saveGroups.request, () => initialState.savedGroups)
+    .handleAction(actions.saveGroups.success, (store, action) => action.payload)
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  .handleAction(actions.saveGroups.success, (store, action) => action.payload)
```